src/asm/x86/ec.rs

```diff
@@ -24,29 +24,66 @@ pub fn update_cdf(cdf: &mut [u16], val: u32) {
 #[target_feature(enable = "sse2")]
 unsafe fn update_cdf_4_sse2(cdf: &mut [u16], val: u32) {
   let nsymbs = 4;
-  let rate = 3 + (nsymbs >> 1).min(2) + (cdf[nsymbs] >> 4) as usize;
+  let rate = 5 + (cdf[nsymbs] >> 4) as usize;
   cdf[nsymbs] += 1 - (cdf[nsymbs] >> 5);
```

Comment:
To further align with dav1d assembly:
```suggestion
  cdf[nsymbs] += (cdf[nsymbs] < 32) as u16;
```

Reply:
Good catch, I overlooked that part. And honestly I overestimated the capabilities of the optimizer -- your solution produces a cleaner asm 👍